include/seqan3/alignment/pairwise/detail/policy_alignment_result_builder.hpp

```diff
@@ -104,9 +104,11 @@ class policy_alignment_result_builder
 
         result_type result{};
 
-        static_assert(!std::same_as<decltype(result.data.id), invalid_t>,
-                      "Invalid configuration. Expected result with id!");
```

Comment:
Why do we not need this static assert anymore?

Reply:
Because before it was required to be set. Now it is optional and depends on the user input.